renovate.json

```diff
@@ -34,15 +40,29 @@
   "packageRules": [
     {
       "groupName": "Maru Support Dependencies",
-      "labels": ["support-deps"],
+      "labels": [
+        "support-deps"
+      ],
       "commitMessageTopic": "support-deps",
-      "packagePatterns": ["*"]
+      "packagePatterns": [
+        "*"
+      ]
     },
     {
       "groupName": "Maru Code Dependencies",
-      "labels": ["code-deps"],
+      "labels": [
+        "code-deps"
+      ],
       "commitMessageTopic": "code-deps",
-      "matchDatasources": ["go"]
+      "matchDatasources": [
+        "go"
+      ]
+    },
+    {
+      "matchPackageNames": [
+        "github.com/pterm/pterm"
+      ],
+      "allowedVersions": "!/-v0.12.80$/"
```

Comment:
Does this need to be 0.12.80+? Or just 0.12.80?

Reply:
@mjnagel at this point I want to specifically not update to `v0.12.80`, which was released in November 2024 and there has been no release since. We know that specific version causes our things to break in some ways. I'm happy to evaluate other `v0.12.8x` or `+` versions when they come though. I assume the filter I have will still show other, non `v0.12.80` versions but please correct me if I'm wrong

Reply:
Yeah correct - if pterm releases a 0.12.81 for example, Renovate would make a new PR for that. But it might be worth adding those as they come rather than globbing them all up now. That way we get a slight nagging to look at the update again every so often.